### ISSUE TITLE
How do you get 3d keypoint?

### ISSUE THREAD

**Issue by curryanswer** [labellson/openpose3d-notebook-visualization#1]
do you use two cameras to get the 3d keypoints? Does it need camera calibration？

**Comment by labellson** [comment]
Yes, I used a stereo system to record the video sequences with a calibrated camera. I remember I developed a module to load stereo images from a folder and to stream from multiple cameras to Openpose. And Openpose already did the projection to the 3D space. I think at that time Openpose was only supporting a specific brand of cameras.

There's a long time since I worked on this. But maybe openpose supports already more sources for multiple cameras.
